openmc/mgxs_library.py

```diff
@@ -333,37 +376,52 @@ def alias(self, alias):
         else:
             self._alias = self._name
 
+    @zaid.setter
+    def zaid(self, zaid):
+        # Check type and value
+        check_type('zaid', zaid, Integral)
+        check_greater_than('zaid', zaid, 0, equality=False)
+        self._zaid = zaid
+
+    @awr.setter
+    def awr(self, awr):
+        # Check validity of type and that the awr value is > 0
+        check_type('awr', awr, Real)
+        check_greater_than('awr', awr, 0.0, equality=False)
+        self._awr = awr
+
     @kT.setter
     def kT(self, kT):
         # Check validity of type and that the kT value is >= 0
-        check_type("kT", kT, Real)
-        check_greater_than("kT", kT, 0.0, equality=True)
+        check_type('kT', kT, Real)
+        check_greater_than('kT', kT, 0.0, equality=True)
```

Comment:
Wait a second, this one is needed since its being passed `True` vice `False`.  Unless you're trying to say that noone is ever going to run a problem at absolute zero, which I agree with, but I guess I technically cant rule it out.